pydfy/models.py

```diff
@@ -51,6 +51,15 @@ class Number(Component):
     template_path: str = field(kw_only=True, default="src/partials/number.html")
 
 
+@dataclass
+class Kpi(Component):
```

Comment:
For this class full capitalization would make sense:
```suggestion
class KPI(Component):
```